diff -Nru aqsis-1.2.0.orig/SConstruct aqsis-1.2.0/SConstruct
--- aqsis-1.2.0.orig/SConstruct	2007-02-18 23:25:17.000000000 +0100
+++ aqsis-1.2.0/SConstruct	2007-03-09 00:40:30.000000000 +0100
@@ -197,7 +197,7 @@
 #	env.AppendUnique(CPPPATH = [env.subst(a)] )
 env.AppendUnique(CPPPATH = ['$boost_include_path'])
 
-env.AppendUnique(CPPDEFINES=[('DEFAULT_PLUGIN_PATH', '\\"' + env.Dir('${PLUGINDIR}').abspath + '\\"')])
+env.AppendUnique(CPPDEFINES=[('DEFAULT_PLUGIN_PATH', '\\"/usr/lib/aqsis/plugins\\"')])
 
 #
 # Enable MP dumping mode if specified
diff -Nru aqsis-1.2.0.orig/platform/default/SConscript aqsis-1.2.0/platform/default/SConscript
--- aqsis-1.2.0.orig/platform/default/SConscript	2007-02-04 11:44:55.000000000 +0100
+++ aqsis-1.2.0/platform/default/SConscript	2007-03-09 01:12:38.512064250 +0100
@@ -128,8 +128,8 @@
 
 # These absolute paths are necessary because there's no good general way of
 # getting a running program's install path on unix-like systems.
-env.AppendUnique(CPPDEFINES=[('DEFAULT_RC_PATH', '\\"' + env.Dir('$SYSCONFDIR').abspath + '\\"')])
-env.AppendUnique(CPPDEFINES=[('DEFAULT_SHADER_PATH', '\\"' + env.Dir('$SHADERDIR').abspath + '\\"')])
+env.AppendUnique(CPPDEFINES=[('DEFAULT_RC_PATH', '\\"/etc/aqsis\\"')])
+env.AppendUnique(CPPDEFINES=[('DEFAULT_SHADER_PATH', '\\"/usr/share/aqsis/shaders\\"')])
 env.Replace(SHLIB_VERSION_SUFFIX = '.%d.%d' %(version.major, version.minor))
 env.AppendUnique(SHLINKFLAGS=['-Wl,-soname=${TARGET.file}.%d' %version.major])
 
